src/index.html

```diff
@@ -12,6 +12,155 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"  
+      method="post">
+      <fieldset class="groups">
+        <legend>Personal information</legend>
+        <label class="form">
+        Surname:
+          <input 
+          type="text"
+          name="Surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            type="text"
            name="Surname"
            autocomplete="off"
          >
```
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

fix code style all over the page